src/render-table.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// {
+//   attributes: { ... },
+//   colgroup: {
+//     attributes: { ... },
+//     cols: [{
+//       attributes: { ... },
+//     }],
+//   },
+//   thead: {
+//     attributes: { ... },
+//     trs: [{
+//       attributes: { ... },
+//       ths: [{
+//         attributes: { ... },
+//         content,
+//       }],
+//     }],
+//   },
+//   tbody: {
```

Comment:
We should use 
```js
tbodies: [{
  ...
}],
```
In order to be aligned with [HTML standard](https://www.w3.org/TR/html/tabular-data.html#the-table-element).